packages/core/addon/templates/components/field-renderer.hbs

```diff
@@ -1,3 +1,6 @@
+{{! eslint is confused about indentation for else-if }}
+{{! template-lint-disable block-indentation}}
+
 {{this.onFieldUpdated}}
```

Comment:
@robustdj this is how we "pull" on the `onFieldUpdated()` getter so that it will act like a `didRecieveAttrs`. There is actually a better pattern to use here which is the `{{did-update}}` render modifier. At some point I'm going to refactor to use that.